marathon_acme/tests/test_acme_util.py

```diff
@@ -0,0 +1,59 @@
+from acme.jose import JWKRSA
+
+from cryptography.hazmat.backends import default_backend
+from cryptography.hazmat.primitives import serialization
+
+from fixtures import TempDir
+
+from testtools import TestCase
+from testtools.matchers import Equals
+
+from twisted.python.filepath import FilePath
+
+from txacme.util import generate_private_key
```

Comment:
You don't need all the empty lines between imports. The usual thing is to have three blocks (stdlib, third-party, local) with an empty line between each block.